components/cam/src/dynamics/se/dp_coupling.F90

```diff
@@ -466,11 +478,18 @@ subroutine p_d_coupling(phys_state, phys_tend,  dyn_in)
 
     if (par%dynproc) then
       if (fv_nphys > 0) then
-
+        call t_startf('fv_phys_to_dyn')
         ! Map FV physics state to dynamics grid
-        call fv_phys_to_dyn(elem,T_tmp(1:nphys_sq,:,:),   &
-                                 uv_tmp(1:nphys_sq,:,:,:),&
-                                 q_tmp(1:nphys_sq,:,:,:))
+        if (se_fv_phys_remap_alg == 0) then
+          call fv_phys_to_dyn(elem,T_tmp(1:nphys_sq,:,:),   &
+               uv_tmp(1:nphys_sq,:,:,:),&
+               q_tmp(1:nphys_sq,:,:,:))
+        else
+          dtime = get_step_size()
```

Comment:
How is dtime used in remap?

Reply:
If Q is a tendency, dt is needed to reconstruct full state for use in the limiter.